src/components/App/App.tsx

```diff
@@ -628,6 +638,20 @@ export default class App extends React.Component<{}, AppState> {
     this.socket.emit('CMD:changeController', data.value);
   };
 
+  setSubtitle = async () => {
+    const files = await openFileSelector();
+    if (!files) {
+      return;
+    }
+    const file = files[0];
+    const reader = new FileReader();
+    reader.addEventListener('load', (event) => {
+      const subData = event.target?.result;
```

Comment:
should probably do a client side check here as well as it's kind of possible for people to accidentally submit a video file here